superset/viz.py

```diff
@@ -896,6 +896,69 @@ def get_data(self):
         return chart_data
 
 
+class BulletViz(NVD3Viz):
+
+    """Based on the NVD3 bullet chart"""
+
+    viz_type = "bullet"
```

Comment:
> lines 903-914 can be deleted, not needed for V2

`viz_type` is still used in the last line of `viz.py` for registration. I cannot get it working if I remove the other lines either. I only know how to turn on V2 for the explore UI. But it does not affect the dashboard UI I think.

Is it okay if I keep the V1 lines for now? I assume they will be mass-deleted at some point. But it's easier for me to debug things in an unknown codebase if I can just follow the examples.